torch_frame/datasets/multimodal_text_benchmark.py

```diff
@@ -482,7 +498,11 @@ def __init__(self, root: str, name: str,
                     col_to_stype[col] = self.text_stype
                 else:
                     col_to_stype[col] = stype
+
+        df = self._pre_transform(df=df, target_col=target_col)
+        col_to_sep = self._dataset_col_to_sep.get(name, '')
```

Comment:
nit: why do we return the empty string by default? When does that happen?

Reply:
When make `col_to_sep`  as none it will raise an error. `col_to_sep` now has some default value `", "` in the dataset.py

Reply:
Why don't you raise an error if `name` is not in `self._dataset_col_to_sep`?

Reply:
This means that there is no multicategorical for that dataset `name`

Reply:
Got it. LGTM as it is now. Thanks for clarifying.